### PR TITLE
Gidk 1dend2 sanity ui issue

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.ts [hunk 1]
```diff
@@ -333,6 +333,7 @@ export class ProformaInvoiceComponent implements OnInit, OnDestroy, AfterViewIni
     public isAddBulkItemInProcess: boolean = false;
     public modalRef: BsModalRef;
     public message: string;
+    public isDropup = true;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.ts:336]: define var type

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 2]
```diff
@@ -9589,6 +9589,25 @@ span#HW_badge_cont:nth-child(3) {
...
 }
+.new-ledger-text-right ul li {
+    ul{
+        li{
+            display: flex;
+            align-items: center;
+        }
+    }
+}
+.total_col bs-tooltip-container.tooltip .tooltip-inner{
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/css/style-2.scss:9600]: avoid snake case

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.ts (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/proforma-invoice/proforma-invoice.component.ts]: Done

File: apps/web-giddh/src/assets/css/style-2.scss (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/assets/css/style-2.scss]: Done

**Review by ravinderwalkover — APPROVED** [review]

**Review by aditi-tiwari17 — APPROVED** [review]

**Pull request merged**